lib/models/channel.js

```diff
@@ -42,6 +42,31 @@ class Channel extends Base {
     return !!this._json.parameters;
   }
 
+  /**
+   * @returns {boolean}
+   */
+  hasServers() {
+    return !!this._json.servers;
+  }
+
+  /**
+   * @returns {String[]}
+   */
+  servers() {
+    if (!this._json.servers) return [];
+    return this._json.servers;
+  }
+
+  /**
+   * @returns {String}
```

Comment:
Please also add parameter annotation:

```suggestion
   * @param {number} index - Index of the server.
   * @returns {String}
```

Reply:
@magicmatatjahu : done